modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -187,7 +187,7 @@ use {{invokerPackage}}\ObjectSerializer;
             } catch (RequestException $e) {
                 throw new ApiException(
                     "[{$e->getCode()}] {$e->getMessage()}",
-                    $e->getCode(),
+                    (int) $e->getCode(),
```

Comment:
Cannot remember which linter(PHPStan or PHP_CodeSniffer) translates:
```diff
--- (int) $e->getCode(),
+++ (int)$e->getCode(),
```
But current [PSR-12 example](https://www.php-fig.org/psr/psr-12/#61-unary-operators) writes type casting with space :
> Type casting operators MUST NOT have any space within the parentheses
> `$intValue = (int) $input;`

So let's consider both coding styles correct. The only thing that current PSR declares is no space within parentheses.